src/webgpu/shader/validation/expression/call/builtin/determinant.spec.ts

```diff
@@ -0,0 +1,93 @@
+const builtin = 'determinant';
+export const description = `
+Validation tests for the ${builtin}() builtin.
+`;
+
+import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../../../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
```

Comment:
nit: This was a bit hard to read. I'd suggest adding a comment

> // Generate a dictionary mapping each matrix type variation (columns,rows, floating point type) to a nontrivial matrix value of that type.

Reply:
Done